### ISSUE TITLE
No build possible with certain packages, need workaround.

### ISSUE THREAD

**Issue by soerendip** [jupyterlab/jupyterlab#8160]
## Description
    jupyter-lab build

>/home/.../miniconda3/envs/jupyterhub/lib/python3.7/site-packages/notebook/services/kernels/kernelmanager.py:19: VisibleDeprecationWarning: zmq.eventloop.minitornado is deprecated in pyzmq 14.0 and will be removed.
    Install tornado itself to use zmq with the tornado IOLoop.
from jupyter_client.session import Session
[LabBuildApp] JupyterLab 1.2.6
[LabBuildApp] Building in /home/.../miniconda3/envs/jupyterhub/share/jupyter/lab
An error occured.
ValueError: Please install nodejs >=6.11.5 before continuing. nodejs may be installed using conda or directly from the nodejs website.
See the log file for details:  /tmp/jupyterlab-debug-s4av70z6.log

## Reproduce

    conda create -n env -c plotly jupyterlab jupyterhub pandas=1 matplotlib scipy scikit-learn python=3.8 plotly_express
    conda activate env
    jupyter-lab build

**Comment by jasongrout** [comment]
Yes, you need to install node version 10 or later, and preferably 12 or later.

**Comment by soerendip** [comment]
Unfortunately, that is not possible within the active conda environment due to version conflicts. Is there a way to install nodejs separately? E.g. with npm just for the build?

**Comment by jasongrout** [comment]
I always install nodejs with conda forge: `conda install -c conda-forge nodejs`

**Comment by jasongrout** [comment]
With jlab 1.2.x, you might be able to use node 8 as well.

**Comment by jasongrout** [comment]
Another way to install and use different versions of node is to use NVM: https://github.com/nvm-sh/nvm
